.pipelines/pipeline.user.windows.buddy.yml

```diff
@@ -9,7 +9,7 @@ build:
     - !!buildcommand
       name: 'Assemble ApplicationInsights-Java JARs'
       command: '.scripts/gradle.cmd'
-      arguments: 'assemble copyLibsToGlobalArtifactsFolder'
+      arguments: 'assemble'
```

Comment:
I think `copyLibsToGlobalArtifactsFolder` is only needed for official release pipeline

Reply:
true, i think it was there before.. so i put it back.. it's fine to remove it.